index.bs

```diff
@@ -140,6 +149,25 @@ The <dfn attribute for="Document">prerendering</dfn> getter steps are to return
 The <dfn attribute for="Document">onprerenderingchange</dfn> attribute is an [=event handler IDL attribute=] corresponding to the <dfn event for="Document">prerenderingchange</dfn> [=event handler event type=].
 
 
+<h2 id="prerender-processing-model">Prerender processing model</h2>
+
+Add a new link type to the "Link types" section of the HTML Standard, replacing the prerender documentation in the Resource Hints standard.
+
+<h3 id="link-rel-prerender">Link type "<dfn data-lt="rel-prerender"><code>prerender</code></dfn>"</h3>
+
+The <a lt="rel-prerender"><code>prerender</code></a> keyword may be used with <a spec=HTML><code>link</code></a> elements. This keyword creates an <a spec=HTML>external resource link</a>. This keyword is <a spec=HTML>body-ok</a>.
+
+To <a lt="process the linked resource" spec=HTML>process this type of linked resource</a> given a <a spec=HTML><code>link</code></a> element <var ignore>el</var>, run these steps:
+
```

Comment:
We should add appropriate times to fetch and process the linked resource. Probably they are UA-defined? Or should we do some of the usual stuff (when it becomes BC-connected, or its relevant attributes change) as something like when the UA should *consider* doing a fetch-and-process?

Reply:
Yeah, I think the usual stuff is best for now, especially since our implementation isn't really UA-defined... at least I think it's taking the existence of the link to be a directive to perform all the fetching and prerendering etc.

Reply:
The wording I used for this matches https://html.spec.whatwg.org/multipage/links.html#link-type-modulepreload which seems pretty standard, and I think matches preload's though it is worded slightly differently